modules/vpc/variables.tf

```diff
@@ -27,6 +27,8 @@ variable "subnetworks" {
   - `create_subnetwork` : Boolean value to control the creation or reading of the subnetwork. If set to `true` - this will create the subnetwork. If set to `false` - this will read a subnet with provided information.
   - `ip_cidr_range` : A string that contains the subnetwork to create. Only IPv4 format is supported.
   - `region` : Region where to configure or import the subnet.
+  - `stack_type` : IP stack type. IPV4_ONLY (default) and IPV4_IPV6 are supported.
+  - `ipv6_access_type` : The access type of IPv6 address. It's immutable and can only be specified during creation or the first time the subnet is updated into IPV4_IPV6 dual stack.
```

Comment:
Let's provide available options as for the argument above.

Reply:
Thanks for the suggestion. Available options added.